cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -23,49 +23,24 @@ For information on validating email addresses, please visit the [input validatio
 
 A key concern when using passwords for authentication is password strength. A "strong" password policy makes it difficult or even improbable for one to guess the password through either manual or automated means. The following characteristics define a strong password:
 
-### Password Length
+- Password Length
+    - **Minimum** length of the passwords should be **enforced** by the application. Passwords **shorter than 8 characters** are considered to be weak ([NIST SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html)). 
+    - **Maximum** password length should not be set **too low**, as it will prevent users from creating passphrases. Typical maximum length is 128 characters.
```

Comment:
But be careful depending on the hashing algorithm used. Bcrypt seems that to have issue with password > 72 positions (see [here](https://security.stackexchange.com/a/39851)). I think that it can be interesting to add a remark about that point.

Reply:
I did not changed this values. We can be consistent with ASVS and put here minimum 12 and make sure that 64 characters or longer passwords are permitted

Reply:
OK sorry.
I think that a remark about the behavior of the hashing algorithm can be useful and justify the definition of a password max size is sometimes needed depending on the algorithm used.

Reply:
Sure we can include note about behavior of the hashing algorithm. What do you think to change minimum and maximin length? Should we aim to be consistent with ASVS?

Reply:
It can be between 8 and 128. 12 is for strong passwords requirements. 8 is the minimum acceptable bar.
A higher end limit is always important to have for all kinds of input, and especially passwords.